.dockleconfig

```diff
@@ -0,0 +1,6 @@
+# This file is allows you to specify a list of files that is acceptable to Dockle
```

Comment:
something that bugs me a bit, maybe we can discuss now or punt to solve in the future, is that the docker scanning is kind of split between template-infra and template-application-flask. the CI is in template-infra, but the configuration files are in the project. it's a little weird then because a project that uses template-application-flask but uses their own custom infra (doesn't use template-infra), will have this `.dockleconfig` file but they won't have any use for it.

maybe for now, can we add something in the README.md that explains that .dockeconfig and .hadolint.yaml are actually used to configure the vulnerability scanning defined in the template-infra repo, and link to the docs in that repo for more information. that way a developer looking at the flask template repo won't be confused and wondering what this file is all about.

Reply:
Makes sense, adding a blurb in the README in the next commit